hr_attendance_ip_check/models/hr_employee.py

```diff
@@ -41,7 +49,9 @@ def _attendance_ip_check(self):
         if allowed_cidrs and not in_cidr:
             raise exceptions.ValidationError(
                 _(
-                    "Not allowed to create new attendance record for %(empl_name)s, the client ip is not in the list of allowed ip networks."
+                    "Not allowed to create new attendance record for"
```

Comment:
add a space after `for`
![image](https://github.com/sonhd91/hr-attendance/assets/8724793/b7569512-a274-4026-bb2a-9349245ee364)

Reply:
Branch updated, please check again